android/src/main/java/guichaguri/trackplayer/logic/services/PlayerService.java

```diff
@@ -36,6 +36,14 @@ public IBinder onBind(Intent intent) {
         }
     }
 
+    @Override
+    public boolean onUnbind(Intent intent) {
+        Log.d(Utils.TAG, "onUnbind");
+        super.onUnbind(intent);
+        manager.onServiceUnbounded();
+        return true;
```

Comment:
Can you leave the result from `super.onUnbind` as the return value?

Reply:
Docs are pretty much clear about that. Default implementation returns false which prevents from call this onUnbind method second time and so on. So we forced to return true here.

https://developer.android.com/reference/android/app/Service.html#onUnbind(android.content.Intent)

Reply:
Fair enough.